components/modal/modal.css

```diff
@@ -0,0 +1,37 @@
+:root{
+    --modal-background-color: #000;
+    --modal-text-color: #fff;
+    --model-content-background-color: #2a9d8f;
+}
```

Comment:
You can create a variable stylesheets and put them inside them.

Reply:
will take care of it